WordPress/Classes/Utility/Universal Links/Routes+Me.swift

```diff
@@ -7,6 +7,13 @@ struct MeRoute: Route {
     let jetpackPowered: Bool = false
 }
 
+struct MeAllDomainsRoute: Route {
+    let path = "/me/domains"
+    let section: DeepLinkSection? = .me
+    let action: NavigationAction = MeNavigationAction.allDomains
+    let jetpackPowered: Bool = true
+}
```

Comment:
@alpavanoglu I think this is not needed, right?

Reply:
Yes I'll remove it